app/consumer.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from app import settings
+from structlog import wrap_logger
+from app.async_consumer import AsyncConsumer
+from app.response_processor import ResponseProcessor
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class Consumer(AsyncConsumer):
+    def on_message(self, unused_channel, basic_deliver, properties, body):
+        logger.info('Received message', delivery_tag=basic_deliver.delivery_tag, app_id=properties.app_id, body=body)
+        try:
+            skip_receipt = (settings.RECEIPT_HOST == "skip")
+            processor = ResponseProcessor(logger, skip_receipt)
```

Comment:
I'm thinking calling this class a ResponseProcessor as well as the Processor that already exists in downstream could get a bit confusing given their different behaviour. Maybe ResponseStorage?